src/course-team/add-user-form/AddUserForm.scss

```diff
@@ -0,0 +1,44 @@
+.add-user-form {
+  display: flex;
+  flex-direction: column;
+  border: .0625rem solid $gray-200;
+  border-radius: .375rem;
+  box-shadow: 0 1px 1px $gray-200;
+  margin-bottom: 20px;
+  background-color: $white;
+
+  .form-title {
+    font-size: 24px;
+    margin-bottom: 30px;
+    font-weight: 600;
+  }
+
+  .form-field {
+    padding: 20px 30px;
+    margin-bottom: 15px;
+
+    .pgn__form-group {
+      margin-bottom: 0;
+    }
+  }
+
+  .form-label {
```

Comment:
At your discretion. My recommendation is to add additional classes for every element that changes styles. There can be many similar classes in the project and it will be very difficult to find this one.